optionlab/engine.py

```diff
@@ -48,9 +48,6 @@ def __init__(self, inputs: Inputs):
         self._n: list[int] = []
         self.action: list[Action | Literal["n/a"]] = []
         self.type: list[StrategyType] = []
-        self._expiration: list[dt.date | int] = (
-            []
-        )  # FIXME: remove? doesn't seem to be used
```

Comment:
I removed it from the engine and everything still worked. so it's being used anywhere

Reply:
Yes, I checked here and I know why. I've changed how days2maturity is computede, probably when I migrated from Cython to pure Python:
self.__days2maturity.append((expirationtmp-self.__startdate).days-ndiscardeddays)

Reply:
No problem, then.